lib/recurly/unique_coupon_list.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+class Recurly_UniqueCouponList extends Recurly_Pager
+{
+
+  public static function createFromUri($uri, $client) {
+    $path = str_replace('/v2', '', parse_url($uri, PHP_URL_PATH));
+    $query = parse_url($uri, PHP_URL_QUERY);
```

Comment:
What do you think about just running `$parts = parse_url($uri)` and just pulling out the relevant pieces?